app/build.gradle

```diff
@@ -103,7 +100,7 @@ android {
 }
 
 def supportVersion = "25.3.1"
-def archVer = "1.0.0-alpha1"
+def archVer = "1.0.0-alpha5"
```

Comment:
Should probably add the doppl dependency now even if we cant test it. You can see it in kevins demo here: https://github.com/doppllib/PartyClickerSample/blob/TheRoom/app/build.gradle

You might need to use the same version (alpha 1) until we can update doppl